lib/bovine/staticinventory.py

```diff
@@ -27,12 +27,15 @@ def __init__(self, root_directory=None):
     self._get_all_hosts()
     self.calc_meta_info()
 
+  def debug_print(self):
+    print(self.inventory)
+
   def _get_all_groups(self):
     '''Walk the groups/ dir, saving all found groups
 
     Then, load all variables, hosts and child groups from each group
     '''
-    groups_directory = self.root_directory + '/groups'
+    groups_directory = self.root_directory + 'groups/'
```

Comment:
this may be the fix, although I can't get tox to work on this stupid mac.  test/test2.py works successfully for me.